docs/products/clickhouse/concepts/strings.rst

```diff
@@ -0,0 +1,62 @@
+
+Aiven for ClickHouse® uses ClickHouse® databases, which can store diverse types of data, such as decimals, boolean data type, or arrays. This article focuses on strings and its usage in ClickHouse. Discover key functions and data type conversions required for working efficiently with strings in ClickHouse.
+
+About strings in ClickHouse
+---------------------------
+
+ClickHouse allows strings of any length. Strings can contain an arbitrary amount of bytes, which are stored and output as-is. The string type replaces the types VARCHAR, BLOB, CLOB, and others from other DBMS. When creating tables, numeric parameters for string fields can be set (for example, TEXT(140)), but are ignored.
+
+ClickHouse supports the following aliases for strings: LONGTEXT, MEDIUMTEXT, TINYTEXT, TEXT, LONGBLOB, MEDIUMBLOB, TINYBLOB, BLOB, VARCHAR, CHAR.
+
+String handling functions
+-------------------------
+
+* `Functions for working with strings <https://clickhouse.com/docs/en/sql-reference/functions/string-functions/>`_
+
+* `Functions for searching in strings <https://clickhouse.com/docs/en/sql-reference/functions/string-search-functions>`_
+
+  .. note::
+    
+    By default, the search is case-sensitive in these functions, but case-insensitive search variants are also available.
+
+* `Functions for searching and replacing in strings <https://clickhouse.com/docs/en/sql-reference/functions/string-replace-functions>`_
+
+* `Functions for splitting and merging strings and arrays <https://clickhouse.com/docs/en/sql-reference/functions/splitting-merging-functions>`_
+
+String conversions
+------------------
+
+Any plain string types can be cast to different types using functions in `Type Conversion Functions <https://clickhouse.com/docs/en/sql-reference/functions/type-conversion-functions>`_.
+
+String and JSON
+---------------
+
+ClickHouse supports a wide range of functions for working with JSON. With specific functions, you can use strings for extracting JSON.
+
+.. seealso::
+    
+    Learn more on `JSON functions in ClickHouse <https://clickhouse.com/docs/en/sql-reference/functions/json-functions/>`_.
+
+.. topic:: Examples
+
+    ``visitParamExtractString(params, name)``
+      Parse the string in double quotes.
+
+    ``JSONExtractString(json[, indices_or_keys]…)``
+      Parse a JSON and extract a string.
+
+    ``toJSONString``
+      Convert a value of any data type to its JSON representation.
+
+More supported data types
+-------------------------
+
+Some other useful data types supported in Aiven for ClickHouse are 
+
+* `Parquet <https://parquet.apache.org/>`_
+    Column-oriented data format, efficient when accessing and reading specific fields and fast when searching for a specific data item
+* `Avro <https://avro.apache.org/>`_
+    Row-based data format (stores all the fields for each record together), used for storing, serializing, and transmitting data over messaging systems (for example, Apache Kafka®) 
+* JSONAsObject
```

Comment:
This should not be in this article:
- strings are a type of data for a column in ClickHouse.
- Parquet/Avro/JSONAsObject are data formats for input/output when selecting or inserting data, they are not data types.

Reply:
Thanks @kmichel-aiven! Two questions:

- Where in the article am I mentioning that "strings are a type of data for a column in ClickHouse" - I'd like to remove that but cannot find it
- I'd remove "Parquet/Avro/JSONAsObject" from this article and perhaps we could add them to the ClickHouse formats article at https://docs.aiven.io/docs/products/clickhouse/reference/supported-input-output-formats - I'd rename this article into more general "ClickHouse-supported formats for data exchange" and include two subchapters: (1) with the existing Kafka info (2) with the Parquet/Avro/JSONAsObject info. We could have this developed in the future. What do you think?

Reply:
By "this", I meant the entire section about  "More supported data types",  the list of items below was just the details of why this shouldn't be in this article.

I wouldn't mix the data exchange formats for ClickHouse-Kafka integration and the other unrelated formats in the same article. One list is specific to Aiven for ClickHouse and the Kafka integration, and the answer to a frequently asked question about that integration, the other is general info about ClickHouse.

We could make a separate article with all the supported data formats outside of the integration, there are approximately 90 data formats though.

Reply:
Understood :-) Thanks - removing the subsection!